### PR TITLE
waiting activity design

### LINKED ISSUE #18: Waiting room activity design
Create Waiting room activity design

### PR DESCRIPTION
**Pull request by agatamrnw**

### CODE REVIEW

File: app/src/main/res/drawable/buttons.xml
Comment (KamilDevelopments) [inline app/src/main/res/drawable/buttons.xml:1]: This file seems to not have any use

File: app/src/main/res/layout/activity_waiting_room_activity.xml [hunk 1]
```diff
@@ -1,29 +1,113 @@
...
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+
+
+        app:layout_constraintEnd_toStartOf="@+id/startGameButton"
+
+        />
+
+    <TextView
+        android:id="@+id/textView"
```
Comment (KamilDevelopments) [inline app/src/main/res/layout/activity_waiting_room_activity.xml]: Name should be meaningful for everyone to understand

File: app/src/main/res/layout/activity_waiting_room_activity.xml [hunk 2]
```diff
@@ -1,29 +1,113 @@
...
+        android:textColor="@color/light_yellow"
+        android:textSize="24sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.174"
+        app:layout_constraintStart_toEndOf="@+id/textView"
+        app:layout_constraintTop_toBottomOf="@+id/waitPlayersTextView"
+        app:layout_constraintVertical_bias="0.026" />
+
+    <androidx.recyclerview.widget.RecyclerView
```
Comment (KamilDevelopments) [inline app/src/main/res/layout/activity_waiting_room_activity.xml:85]: RecyclerView is missing an id

File: app/src/main/res/layout/activity_waiting_room_activity.xml [hunk 3]
```diff
@@ -1,29 +1,113 @@
...
+        android:layout_width="168dp"
+        android:layout_height="87dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginEnd="132dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView3" />
+
+    <TextView
+        android:id="@+id/textView4"
```
Comment (KamilDevelopments) [inline app/src/main/res/layout/activity_waiting_room_activity.xml]: Same as above

File: app/src/main/res/layout/activity_waiting_room_activity.xml [hunk 4]
```diff
@@ -1,29 +1,113 @@
...
+        android:text="@string/name_of_room"
+        android:textColor="@color/light_yellow"
+        android:textSize="24sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/waitPlayersTextView"
+        app:layout_constraintVertical_bias="0.027" />
+
+    <TextView
+        android:id="@+id/textView3"
```
Comment (KamilDevelopments) [inline app/src/main/res/layout/activity_waiting_room_activity.xml]: Same as above

**Review by KamilDevelopments — CHANGES_REQUESTED** [review]
Also you should add description to your pull request.

**Review by kestivvi — DISMISSED** [review]
Looks good to me

**Comment by agatamrnw** [comment]
added missing ids, deleted unnecessary file and renamed the style, let me know whether colors's names are questionable as well

File: app/src/main/res/layout/activity_waiting_room_activity.xml [hunk 5]
```diff
@@ -73,29 +65,31 @@
...
         android:layout_width="168dp"
         android:layout_height="87dp"
         android:layout_marginTop="20dp"
         android:layout_marginEnd="132dp"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/textView3" />
+        app:layout_constraintTop_toBottomOf="@+id/playersTextView" />
 
     <TextView
         android:id="@+id/textView4"
```
Comment (KamilDevelopments) [inline app/src/main/res/layout/activity_waiting_room_activity.xml]: This one should be also changed

**Review by KamilDevelopments — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_waiting_room_activity.xml (hunk 5 above)
Comment (KamilDevelopments) [reply app/src/main/res/layout/activity_waiting_room_activity.xml]: textView4

**Comment by KamilDevelopments** [comment]
Also there is a typo in RecyclerView id

**Review by KamilDevelopments — DISMISSED** [review]
Lookin good now

**Review by KamilDevelopments — APPROVED** [review]

**Pull request merged**